sdks/java/core/src/test/java/org/apache/beam/SdkCoreApiSurfaceTest.java

```diff
@@ -30,7 +30,7 @@
  * API surface verification for {@link org.apache.beam}.
  */
 @RunWith(JUnit4.class)
-public class SdkCoreApiSurfaceTest extends ApiSurfaceVerification {
+public class SdkCoreApiSurfaceTest extends ApiSurfaceVerificationTest {
 
   @Override
   protected ApiSurface apiSurface() throws IOException {
```

Comment:
I was able to drop quite a few packages here and it looked a bit suspicious. 
Are all these packaged no longer exposed?